docs/source/api-basic.rst

```diff
@@ -0,0 +1,177 @@
+.. _api-basic-chapter:
+
+###########################
+API Reference Point - Basic
+###########################
+
+============
+Requirements
+============
+
+
+Before using this chapter's classes, you need to be able to import the following module.
+
+.. code-block:: python
+
+   import easygopigo3
+
+If you have issues importing these 2 modules, then make sure that:
+
+   * You've followed the steps found in :ref:`Getting Started <getting-started-chapter>` guide.
```

Comment:
Only making this comment because you use "You have" everywhere else and suddenly use the contraction here.